asset/asset.json

```diff
@@ -1,5 +1,5 @@
 {
     "name": "file",
-    "version": "2.8.2",
+    "version": "2.8.3",
```

Comment:
with semver, a patch is just an internal bug fix, however this is a change of the api and contracts, this should be a minor update